web/frontend/src/app/component/subset-details/subset-details.component.html

```diff
@@ -44,19 +44,19 @@
                [lazy]="true" (onLazyLoad)="onLazyLoadData($event)" class="table table-striped table-bordered">
         <ng-template pTemplate="header">
           <tr>
-            <th class="w-10">Code</th>
-            <th class="w-10">Name</th>
-            <th class="w-40">Synonyms</th>
-            <th class="w-40">Definitions</th>
+            <th class="w-10">NCIt Concept Code</th>
+            <th class="w-10">NCIt Preferred Term</th>
+            <th class="w-40">NCIt Synonyms</th>
+            <th class="w-40">NCIt Definition</th>
           </tr>
         </ng-template>
         <ng-template pTemplate="body" let-value let-i="rowIndex">
           <tr>
-            <td class="w-10">{{ value.code }}</td>
-            <td class="w-10"><a [routerLink]="[urlBase, value.code ]" target='urlTarget'>{{ value.name }}</a></td>
+            <td class="w-10"><a [routerLink]="[urlBase, value.code ]" target='urlTarget'>{{ value.code }}<i class="fa fa-external-link" style="margin-left:0.2em;"></i></a></td>
+            <td class="w-10">{{ value.name }}</td>
             <td class="w-40">
               <span *ngFor="let syn of synonymSources[i] | keyvalue">
-                  <p> {{syn.key}}: {{syn.value}} </p>
+                  <p> {{syn.value}} </p>
```

Comment:
In the next section, we ONLY want to show the NCI definition(s).  These are likely the same semantics as the synonyms, with a source=NCI or without a source (though I think they always have a source of NCI)